library/src/main/java/com/getstream/sdk/chat/adapter/viewholder/message/configurators/UsernameAndDateConfigurator.kt

```diff
@@ -41,7 +41,7 @@ internal class UsernameAndDateConfigurator(
 
         if (style.isMessageDateShow) {
             binding.tvMessagedate.isVisible = true
-            binding.tvMessagedate.text = TIME_DATEFORMAT.format(messageItem.message.createdAt)
+            binding.tvMessagedate.text = TIME_DATEFORMAT.format(messageItem.message.createdAt!!)
```

Comment:
You may have a NullPointerException here, as createdAt is optional. Maybe:
```
binding.tvMessagedate.text = messageItem.message.createdAt.let(TIME_DATEFORMAT::format)
```

To avoid the NPE.

Reply:
We've always assumed that this exists, there was never a `null` check here previously. I'd rather have this very obviously break if it somehow ends up being a `null` value, as opposed to just silently not updating the date text.

The property itself is nullable because of how we have serialization set up, kind of an unfortunate side effect of rolling with Gson.

Reply:
I think it is nullable because local messages that has not been sent yet doesn't have `createAt` Date.
There is a related task about that already in our jira: https://stream-io.atlassian.net/browse/CAS-224